pkg/scripts/node.go

```diff
@@ -18,13 +18,13 @@ package scripts
 
 const (
 	drainNodeScriptTemplate = `
-export KUBECONFIG=/etc/kubernetes/admin.conf
-sudo kubectl drain {{ .NODE_NAME }} --ignore-daemonsets --delete-local-data
+sudo KUBECONFIG=/etc/kubernetes/admin.conf \
+    kubectl drain {{ .NODE_NAME }} --ignore-daemonsets --delete-local-data
```

Comment:
Can we make it consistent between addons and node scripts and use either flag or the environment variable?

Reply:
👍 done